tsconfig.json

```diff
@@ -25,7 +25,7 @@
   },
   "include": [
     "./src"
-  ],
+, "__tests__/components/Button/helpers/helper.specs.tsx"  ],
```

Comment:
Why?

Reply:
Resolved in 8fd92b4e11cfc7f5bbfd723370608912e53f3890